workbench-core/common-ui/src/common/splitPanel.tsx

```diff
@@ -27,8 +27,6 @@ export const getPanelContent = (items: any, itemType: string): any => {
     };
   }
 
-  const item = items[0];
```

Comment:
Non related to PR, since we are not showing the cost panel this functions are not being used,
but this functions are very specific to that cost panel, is this worth moving to common-ui?
any thoughts on this @SanketD92  @nguyen102 ?